play-services-core/src/main/kotlin/org/microg/gms/accountaction/ErrorResolver.kt

```diff
@@ -128,8 +128,11 @@ fun <T> Resolution.initiateFromBackgroundBlocking(context: Context, account: Acc
             }
             return null
         }
-        Reauthenticate -> {
+        is Reauthenticate -> {
             Log.w(TAG, "Your account credentials have expired! Please remove the account, then sign in again.")
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
@DaVinci9196
@mar-v-in
[This](https://github.com/microg/GmsCore/pull/2865/files#diff-b9b9347e3a0cd101b758bf9a6e34d55e490fde3821086bb48e2e29978d3a8408L5) is another remnant before the decision of:
- For fake data (org.microg.gms.profile.Build) import all
- For real data (android.os.Build)  import just the specific fields